README.adoc

```diff
@@ -0,0 +1,17 @@
+= RISC-V SBI specification
+
+SBI (Supervisor Binary Interface) is the only interface between M-Mode and
+S-Mode on Unix-class platforms. It allows S-Mode to execute some privileged
+operations by using the ecall mechanism.
```

Comment:
I think it's better to write in a generic way:
SBI (Supervisor Binary Interface) is an interface between Supervisor Execution Environment (SEE) and the supervisor. The M-Mode/S-mode and HS/VS mode can be described as an example to this.